app/models/institution.rb

```diff
@@ -17,4 +18,12 @@ def obfuscated_short_name
   def total_receivers
     receivers.count
   end
+
+  def total_receivers_with_gift
+    matches.received.count
+  end
+
+  def total_receivers_without_gift
+    receivers.count - matches.received.count
+  end
```

Comment:
It's complicating stuff, this is just a quick "hack" (i don't think this is a hack though) so the Dashboard can render these as `Institution` attributes get it?

Reply:
These show on the `index` and `show` views of the admin dashboard. Nelson's idea.

Reply:
I'm not comfortable with this at all. You can define custom values on administrate IIRC. In here you are adding stats/analytics data to a given institution. I really fell like we should separate this. Regardless of it being a hack or not.

Reply:
Only way would be to extract to a value object, but still you would have to have these methods here, so that they can appear on the Institution dashboard.

Reply:
These are the stats for a SINGLE INSTITUTION get it, it's not global, so extracting and creating a new dashboard is not an option. I don't know how to "separate" this really.

Reply:
Yes, I know. I'm approving this out of sheer lack of time but this is not the correct approach and should be fixed ASAP.

Reply:
Well, then click Approve. Put a "remind me" on this :joy: